build_defs/kotlin.bzl

```diff
@@ -0,0 +1,25 @@
+"""Arcs Kotlin WASM rules.
+
+Rules are re-exported in build_defs.bzl -- use those instead.
+"""
+
+load("//build_defs/kotlin_native:build_defs.bzl", "kt_wasm_binary", "kt_wasm_library")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+_ARCS_KOTLIN_LIBS = ["//src/wasm/kotlin:arcs_wasm"]
+
+def arcs_kt_library(name, srcs = [], deps = []):
+    """Declares kotlin library targets for Kotlin particle sources."""
+    kt_wasm_library(
+        name = name,
+        srcs = srcs,
+        deps = _ARCS_KOTLIN_LIBS + deps,
+    )
+
+def arcs_kt_particle(name, srcs = [], deps = []):
```

Comment:
I don't like this change, because it makes 1 particle per wasm file, with the current override of the native runtime, a large recipe with many small particles would end up with a bunch of 500k-1m wasm files because of duplicate runtimes.

Also, we very much want multiplatform compilation to work. Write one Particle, use 1 build rule, deploy to either Dev or Prod experience.

Reply:
You can bundle multiple particle implementations inside a single arcs_kt_particle. It's just a binary. I could rename it to arcs_kt_binary if you think that's clearer?

Reply:
You mean, you can put particles in arcs_kt_library, and then make arcs_kt_particle dep on them? Sure, but if that's the case, we should call it arcs_kt_binary then as you say.

Reply:
Renamed arcs_kt_particle -> arcs_kt_binary